src/app/Balloon.App.ClamAv/src/lib/Cli.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Balloon
+ *
+ * @author      Raffael Sahli <sahli@gyselroth.net>
+ * @copyright   Copryright (c) 2012-2017 gyselroth GmbH (https://gyselroth.com)
+ * @license     GPLv3 https://opensource.org/licenses/GPL-3.0
+ */
+
+namespace Balloon\App\ClamAv;
+
+use \Balloon\Filesystem\Node\File;
+use \Balloon\App\AbstractApp;
+use \Balloon\App\AppInterface;
+
+class Cli extends AbstractApp
+{
+    const FILE_INFECTED = 0;
+    const FILE_OK = 1;
+
+    /**
+    * Socket
+    *
+    * @var string
+    */
+    protected $socket = 'unix:///var/run/clamav/clamd.ctl';
+
+    /**
+    * Maximum Stream Size
+    *
+    * @var int
+    */
+    protected $maxStreamSize = 26214400;
+
+    /**
+     * Init
+     *
+     * @return bool
+     */
+    public function init(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Set options
+     *
+     * @var Iterable $config
+     */
+    public function setOptions(?Iterable $config=null): AppInterface
+    {
+        if($config === null) {
+            return $this;
+        }
+
+        foreach($config as $option => $value) {
+            switch($option) {
+                case 'socket':
+                    $this->socket = (string)$value;
+                    break;
+                case 'maxStreamSize':
+                    $this->maxStreamSize = (int)$value;
+                    break;
+            }
+        }
+
+        return $this;
+    }
+
+    /**
+     * Scan file
+     *
+     * @param  File $file
+     * @return int
+     */
+    public function scan(File $file): int
+    {
+        if ($file->getSize() > $this->maxStreamSize) {
```

Comment:
methods and variables must use underscore, see CONTRIBUTING.md (code policy)

Reply:
sry of course I meant properties, methods are fine with camelCase

Reply:
variables, class properties and config values => underscore

Reply:
fixed in #70b21c2